src/main.rs

```diff
@@ -117,6 +117,9 @@ struct Args {
         help = "Prefix for Redis keys"
     )]
     redis_key_prefix: String,
+
+    #[arg(long, env, value_delimiter = ',', help = "API keys to allow")]
+    api_keys: Vec<String>,
```

Comment:
This configuration will actually parse the empty `--api-keys` flag or `API_KEYS=` env var into a Vec containing a single empty string.
You'll probably want to filter out all empty strings after applying the parser/delimiter, like so:
```rust
let api_keys = args.api_keys.into_iter().filter(|key| !key.is_empty()).collect()
```